src/lib/higher-order-helpers.js

```diff
@@ -0,0 +1,5 @@
+export function getDisplayName(WrappedComponent) {
```

Comment:
Nice for debugging!

Reply:
I think you're reinventing the wheel here no? Isn't there a function like this in React, that's what I saw in the docs, let me check.

Reply:
https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging

Reply:
Okay I submit :P, just seems weird everyone reimplements this for every author of HOCs